app/src/main/java/app/revanced/manager/util/Util.kt

```diff
@@ -165,4 +172,22 @@ fun String.relativeTime(context: Context): String {
     } catch (e: DateTimeParseException) {
         return context.getString(R.string.invalid_date)
     }
+}
+
+@Composable
+fun LazyListState.isScrollingUp(): Boolean {
+    var previousIndex by remember(this) { mutableIntStateOf(firstVisibleItemIndex) }
+    var previousScrollOffset by remember(this) { mutableIntStateOf(firstVisibleItemScrollOffset) }
```

Comment:
Can these be moved inside the `remember` block?

Reply:
Marking as resolved because this breaks the functionality by forgetting the state once scrolling ends

Reply:
Nevermind, done